lib/jasmine-2.2.0/boot.js

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
I know we haven't gone into it too much in the course, but it is quite a good idea to use bower to install things like jasmine - this allows you to then not have to check in files like this - it works the same way as Bundler but for client-side JS files